### PR TITLE
Added ciphers comparison table to README

### PR DESCRIPTION
**Pull request by Avatat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,5 +40,19 @@ The benchmark is written in C language.
...
+| ChaCha20-Poly1305 | NSS | MPL 2|  |
+| AES-128-OCB | OpenSSL | Apache | Patented |
+| AES-256-GCM | OpenSSL | Apache |  |
+| ChaCha20-Poly1305 | OpenSSL | Apache |  |
+| AEGIS-128L | libsodium | ISC |  |
+| AEGIS-256 | libsodium | ISC |  |
+| AES-256-GCM | libsodium | ISC | [Requires SSSE3, AES-NI and CLMUL](https://doc.libsodium.org/secret-key_cryptography/aead/aes-256-gcm#limitations) |
+| ChaCha20-Poly1305 | libsodium | ISC |  |
+| AES-256-GCM | wolfCrypt | GPLv2 |  |
+| ChaCha20-Poly1305 | wolfCrypt | GPLv2 |  |
```
Comment (davidebeatrici) [inline README.md]: Should we perhaps add "Optimizations disabled by default"?

wolfSSL/wolfssl#2691

Comment (Avatat) [reply README.md]: Done!

File: README.md [hunk 2]
```diff
@@ -40,5 +40,19 @@ The benchmark is written in C language.
...
+| Algorithm | Library | License | Limitations |
+| --------- | ------- | ------- | ----------- |
+| AES-256-GCM | NSS | MPL 2 |  |
+| ChaCha20-Poly1305 | NSS | MPL 2|  |
+| AES-128-OCB | OpenSSL | Apache | Patented |
+| AES-256-GCM | OpenSSL | Apache |  |
+| ChaCha20-Poly1305 | OpenSSL | Apache |  |
+| AEGIS-128L | libsodium | ISC |  |
+| AEGIS-256 | libsodium | ISC |  |
+| AES-256-GCM | libsodium | ISC | [Requires SSSE3, AES-NI and CLMUL](https://doc.libsodium.org/secret-key_cryptography/aead/aes-256-gcm#limitations) |
```
Comment (davidebeatrici) [inline README.md]:
```suggestion
| AES-256-GCM | libsodium | ISC | [Requires SSSE3 + `aesni` and `pclmul` instructions](https://doc.libsodium.org/secret-key_cryptography/aead/aes-256-gcm#limitations) |
```

**Review by davidebeatrici — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Avatat) [reply README.md]: Fixed!

**Review by davidebeatrici — APPROVED** [review]

**Review by davidebeatrici — APPROVED** [review]

**Pull request merged**